asteroid/models/zenodo.py

```diff
@@ -76,14 +76,14 @@ def change_metadata_in_deposition(self, dep_id, metadata):
                 `r = create_new_deposition(); dep_id = r.json()['id']`
             metadata (dict): Metadata dict.
 
-        Examples:
-            metadata = {
-                'title': 'My first upload',
-                'upload_type': 'poster',
-                'description': 'This is my first upload',
-                'creators': [{'name': 'Doe, John',
-                              'affiliation': 'Zenodo'}]
-            }
+        Examples
+            >>> metadata = {
+            >>>     'title': 'My first upload',
```

Comment:
all prefixes except the first should be `...`

Reply:
Because it's one instruction?

Reply:
Yep! I assume you wanted it to look like the Python repl.